src/lib/utils/date.ts

```diff
@@ -0,0 +1,9 @@
+import dateFormat from 'dateformat';
+
+export function formatDate(date: string): string {
+	return dateFormat(date, 'UTC:dd/mm/yyyy');
+}
+
+export function formatTime(date: string): string {
```

Comment:
Hi @grmbyrn I think the argument should be renamed to `time`.

Reply:
I made that change to the argument and have started looking into how to add unit tests.